STR-Ontology/OwlModels/configuration.rdf

```diff
@@ -2,7 +2,7 @@
     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
     xmlns="http://com.ge.research.sadl.configuration#">
   <Category rdf:about="http://com.ge.research.sadl.configuration#ProjectDependencies">
-    <pDependsOn>file:/RACK/RACK-Ontology</pDependsOn>
+    <pDependsOn>file:/Users/emertens/Source/arcos/RACK/RACK-Ontology</pDependsOn>
```

Comment:
I believe the rdf files are generated by sadl, should/could they be removed from the repo.  I say this because I noticed they have a user specific path (i.e. /Users/emertens...)